spread-tests/regression/lp-1599891/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Regression check for https://bugs.launchpad.net/snap-confine/+bug/1599891
+# This is blacklisted on debian because debian doesn't use apparmor yet
+systems: [-debian-8]
+execute: |
+    snap_confine=$(realpath $(which ubuntu-core-launcher))
+    echo "Seeing that snap-confine is in $snap_confine"
+
+    echo "I also see a corresponding apparmor profile"
+    grep -F -q "$snap_confine (enforce)"  "/sys/kernel/security/apparmor/profiles"
```

Comment:
This test is fine but and makes sure there is an enforce mode profile but is a bit brittle in the face of renames or snapd/snap-confine/snap-run miscoordinated updates. I'd still prefer we are testing enforcement in the manner I described since it will ensure that snap-confine is running under confinement when invoked from the snapd/snap-run that is installed.